tests/test_cv_splitter.py

```
@@ -0,0 +1,236 @@
+        # Group 1: 小差异 (100, 110, 120) - CV 较低
+        # Group 2: 大差异 (50, 500) - CV 很高
+        # 当限制为 2 组时，应该优先分割 CV 大的
+        pairs = [
+            (100.0, "A1"),
+            (110.0, "A2"),
+            (120.0, "A3"),
+            (50.0, "B1"),
+            (500.0, "B2"),
+        ]
+
+        # 不限制组数，看看会分成几组
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)